moonlight.conf

```diff
@@ -40,6 +40,9 @@
 ## Play audio on host instead of streaming to client
 #localaudio = false
 
+## Send quit app request to remote after quitting session
+#quitappafter = true
```

Comment:
Like the rest of the example configuration file, it should contain the default value ;) . So this should be `quitappafter = false`

Reply:
Ah,  I was thinking on that too :)  Will fix it now. Thanks!

Reply:
Done!